pyramid_jinja2/__init__.py

```diff
@@ -128,9 +121,9 @@ def caller_package(self, excludes=()):
 
 
 class SmartAssetSpecLoader(FileSystemLoader):
-    '''A Jinja2 template loader that knows how to handle
+    """A Jinja2 template loader that knows how to handle
     asset specifications.
-    '''
+    """
```

Comment:
In the future, please isolate automated formatting of code to separate PR and as a final step. No one wants to review a bajillion changes to find the ones that matter.

If it is at all possible, please revert any formatting changes for now.

Reply:
Yes I expected the PR to be too big. But to pass the lint test, you have to lint the code.
I'm going to separate cookiecutter and the black formatting in different PRs.

Reply:
Thank you! I appreciate it.